cloudformation/main.yaml

```diff
@@ -1,11 +1,43 @@
 Resources:
+  # Database2
+  EmailTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      TableName: 'Emails'
```

Comment:
While technically true, wouldn't this be more like 'registrations' or something like that since it contains the email and dropbox config?

Reply:
Yep - will rename once we confirm the expected behavior!